gameSource/languages/help_English

```diff
@@ -11,13 +11,17 @@
 title$MY PHRASEBOOK$
 IF I SAY...
 space$0.25f
-sub$'NO BB'$
```

Comment:
Again, how come you've replaced the 'NO BB' part? This is still a valid phrase.

Reply:
This are the phrase that most player will first learn in game, because it is in the help screen. Thus, it is better if it is a polite and non informal phrase.
NO BB can be considered rude and is informal. That phrase will still be valid in game, and probably will continue being the most used phrase it will just not be the phrase on the help screen.